### ISSUE TITLE
Potential error in `process-cookies`

### ISSUE THREAD

**Issue by FrankRuben** [donaldsonjw/fastcgi#1]
I might not see something obvious, but the implementation of `process-cookies` in `fastcgi.scm` seems to be incorrect:
Shouldn't the line read `(loop (cdr lst))` instead of `(loop (cdr cookies))`?

**Comment by donaldsonjw** [comment]
You are absolutely correct. I pushed this fix along with some updates I made in 2021 to support building under newer versions of Bigloo. Those changes also remove the dependency on the pthread-extra library.

I am not actively using the fastcgi library at the moment but am willing to assist if you are looking to use it.

I appreciate the bug report.

Thanks,
Joseph
